docs/application/native/guides/ui/dali/ui-components-n.md

```diff
@@ -24,6 +24,7 @@ The following table lists the available UI components.
 | [ImageView](./imageview-n.md)         | An image view that renders an image.     | `ImageView`                              |
```

Comment:
The description text in the table needs modification. Suggestion are given. 

A **push button** that can be pressed. A **check box** that can be selected or cleared. A **radio button** where only one option can be selected.
This control is a layout model that allows responsive elements within a container, to automatically arrange the elements to different screen sizes or devices.
This control is used to render an image.
This control is used to enable render item sets in a scrollable layout.
This control is used to enable static 3D content display capability.
This control gives you a recursive update on the progress of an ongoing operation.
This control is used to enable the scroll function in the UI.
This control allows you to select a value within a predefined range.
This control is used to enable render a grid layout of text. 
This control is used to enable render a short text string. 
This control is used to enable a field for one-line of editable text. 
This control is used to enable a field for multiple lines of editable text.
This control is used to enable video playback.

Reply:
Changed the description similar to your comment.